src/hot-keys/RaceSpecificHotkeys/hotkeyData/human/heroes.ts

```diff
@@ -28,7 +28,7 @@ const heroes =
                 Ability.HeroAbility('Storm Bolt', 'btnstormbolt', 'ahtb', 'T', []),
                 Ability.HeroAbility('Thunder Clap', 'btnthunderclap', 'ahtc', 'C', []),
                 Ability.Aura('Bash', 'pasbtnbash', 'ahbh', 'B', []),
-                Ability.HeroAbility('Avatar', 'btnavatar', 'ahav', 'V', []),
+                Ability.UnhotkeyAbility('Avatar', 'btnavatar', 'ahav', 'V', 'V', true),
```

Comment:
can you unhotkey avatar? oh wow xD

Reply:
I have absolutely no idea what it means (I mean, can you actually cancel it early??? wtf??) but the documentation says so...

`// Avatar
[AHav]
Hotkey=V
Unhotkey=V
Researchhotkey=V
`
Obviously I'm taking the official documentation at face value 👼 so there's that...